stable/artifactory-ha/templates/metrics-service.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "artifactory-ha.fullname" . }}-metrics
+  labels:
+    app: {{ template "artifactory-ha.name" . }}
+    chart: {{ template "artifactory-ha.chart" . }}
+    component: metrics
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  type: ClusterIP
```

Comment:
You should give the option to change the service type. Please use `.Values.metrics.service.type` here instead of hard-coding `ClusterIP`

Reply:
fixed